campaigns/guides/bandwidth10dlcCampaignImportApiGuide.md

```diff
@@ -0,0 +1,132 @@
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+```
+
+### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/imports?page=1&size=2
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<LongCodeImportCampaignsResponse>
+    <ImportedCampaigns>
+        <ImportedCampaign>
+            <CampaignId>CCOPVEY</CampaignId>
+            <Description>Test 9</Description>
+            <MessageClass>Campaign-E</MessageClass>
+            <CreateDate>2021-03-18T12:50:45Z</CreateDate>
+            <Status>ACTIVE</Status>
+        </ImportedCampaign>
+        <ImportedCampaign>
+            <CampaignId>CCOPRTM</CampaignId>
+            <Description>Test 10</Description>
+            <MessageClass>Campaign-E</MessageClass>
+            <CreateDate>2021-03-19T09:36:18Z</CreateDate>
+            <Status>ACTIVE</Status>
+        </ImportedCampaign>
+    </ImportedCampaigns>
+</LongCodeImportCampaignsResponse>
+```
+
+{% endextendmethod %}
+
+## Import campaign
+
+{% extendmethod %}
+
+#### Request URL
+<code class="post">POST</code>`https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/imports`
+
+| Request Body               | Mandatory | Description                                                                                                                |
+|:---------------------------|:----------|:---------------------------------------------------------------------------------------------------------------------------|
+| `CampaignId`               | Yes       | Campaign ID generated by TCR                                                                                               |
+
+### POST Imports
+
+{% sample lang="http" %}
+
+```http
+POST https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/imports HTTP/1.1
+Content-Type: application/xml; charset=utf-8
+Authorization: Basic dXNlcm5hbWU6cGFzc3dvcmQ=
+
+<ImportedCampaign>
+    <CampaignId>CJEUMDK</CampaignId>
+</ImportedCampaign>
+
+```
+
+### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/xml
+Location: https://dashboard.bandwidth.com/api/accounts/{accountId}/campaignManagement/10dlc/campaigns/imports
+
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<LongCodeImportCampaignsResponse>
+    <ImportedCampaign>
+        <CampaignId>CJEUMDK</CampaignId>
+        <Description>Test 9</Description>
+        <MessageClass>Campaign-E</MessageClass>
+        <CreateDate>2021-03-18T12:50:45Z</CreateDate>
+        <Status>ACTIVE</Status>
+    </ImportedCampaign>
+</LongCodeImportCampaignsResponse>
+```
+
+{% endextendmethod %}
+
+## Assign a campaign to a TN
+For more info on TNs, please see [Number Management](../../numbers/about.md)<br/>
+For more info on assigning campaigns to TNs, please see [Advanced Ordering](../../numbers/guides/advancedOrdering.md)
```

Comment:
@ajrice6713 @hamermike is there a reason this `advancedOrdering.md` file was removed from the docs? Or was it just never added? These links are correct but because we don't have the advanced ordering section in SUMMARY.md, they're not resolving. Just wanted to double check before saying we can re-add them

Reply:
Actually AJ says this guide `onDemandNumberSearchAndOrder.md` is better to use